resources/js/app.js

```diff
@@ -7,16 +7,35 @@
 
 require('./bootstrap');
 
-window.Vue = require('vue');
+// window.Vue = require('vue');
 
 /**
  * Next, we will create a fresh Vue application instance and attach it to
  * the page. Then, you may begin adding components to this application
  * or customize the JavaScript scaffolding to fit your unique needs.
  */
 
-Vue.component('example-component', require('./components/ExampleComponent.vue'));
+// Vue.component('example-component', require('./components/ExampleComponent.vue'));
 
-const app = new Vue({
-    el: '#app'
-});
+// const app = new Vue({
+//     el: '#app'
+// });
+
+function ignoreProject(id, ignore = true) {
+	var counterElement = document.getElementById('projectCounter');
+	var count = parseInt(counterElement.textContent);
+	
+	var route = (ignore) ? 'ignore' : 'unignore';
+
+	axios.patch('/' + route + '/' + id).then(function(response) {
+		if (response?.status === 200) {
+			document.getElementById('project_' + id).style.display = 'none';
+			counterElement.innerHTML = (count-1);
+		}
+		else {
+			alert('Sorry, something went wrong!');
+		}
+	});
+}
+
+global.ignoreProject = ignoreProject;
```

Comment:
Please re-add the newline at the bottom of the file.

Reply:
Not quite sure if I understand what you mean here?

Reply:
If you're refering to the new unignoreProject function, thats what I did

Reply:
There was a newline at the end of this file but it's not a big deal 😄 

![image](https://user-images.githubusercontent.com/1141514/96071924-ef05a400-0e57-11eb-8ebb-30bf5997ede5.png)